pkg/k8s/configmap.go

```diff
@@ -1,22 +1,29 @@
 package k8s
 
 import (
+	apiv1alpha1 "github.com/percona/percona-server-mysql-operator/api/v1alpha1"
+	"github.com/percona/percona-server-mysql-operator/pkg/naming"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func ConfigMap(name, namespace, filename, data string) *corev1.ConfigMap {
-	return &corev1.ConfigMap{
+func ConfigMap(cr *apiv1alpha1.PerconaServerMySQL, name, filename, data string) *corev1.ConfigMap {
+	cm := &corev1.ConfigMap{
 		TypeMeta: metav1.TypeMeta{
 			APIVersion: "v1",
 			Kind:       "ConfigMap",
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
-			Namespace: namespace,
+			Namespace: cr.Namespace,
 		},
 		Data: map[string]string{
 			filename: data,
 		},
 	}
+	if cr.CompareVersion("0.12.0") >= 0 {
+		cm.Labels = naming.Labels(name, cr.Name, "percona-server", "database")
```

Comment:
isn't this function used for all config maps? is it okay to set `database` as component in every config map?